server/src/main/java/io/spine/server/integration/AbstractChannelObserver.java

```diff
@@ -41,7 +41,7 @@
 
 /**
  * Base routines for the {@linkplain Subscriber#addObserver(StreamObserver)}
- * subscriber observers}.
+ * subscriber observers.
```

Comment:
This change needs to remove either preceeding `}` so that `linkplain` works, or, make `likplain` be just `link`.